theia-swift-docker/Dockerfile

```diff
@@ -4,7 +4,7 @@ RUN npm install -g vsce
 RUN git clone --depth 1 https://github.com/apple/sourcekit-lsp
```

Comment:
Should the commit be pinned somehow? I assume this is how it broke in the first place?

Reply:
I think we could check out a specific tag and it’d be much more stable.

Reply:
We're basically pulling/building their master branch, so it's possible we'll encounter some temporary issues. However any issue we see should not last very long. I am comfortable with that, so long as it's now-maintenance for us.